client/core/core.go

```diff
@@ -1366,8 +1366,9 @@ func (c *Core) dexConnections() []*dexConnection {
 // exchangeMap creates a map of *Exchange keyed by host, including markets and
 // orders.
 func (c *Core) exchangeMap() map[string]*Exchange {
-	infos := make(map[string]*Exchange, len(c.conns))
```

Comment:
It was the `len(c.conns)`.  I was just gonna remove the pre-alloc, but it's just as easy to grab the slice first and use it in the `range`